.gitignore

```diff
@@ -1,7 +1,8 @@
 .idea
 node_modules
 npm-debug.log
-public
```

Comment:
This should have covered it—not that your change is problematic, just wondering what triggered the change.